app/views/home/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Hello World</h1>
+<button type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="top" title="Tooltip on top">
```

Comment:
add meaningful content, like Welcome to Employee Tracking System, otherwise remove everything that is added for testing